src/main/kotlin/com/learnspigot/bot/manager/ForumManager.kt

```diff
@@ -122,6 +122,10 @@ class ForumManager(private val bot: JDA, private val datastore: Datastore, priva
                     val profile: UserProfile = datastore.findUserProfile(it)
                     profile.addRep(channel, channel.owner!!, leaderboardManager, event.guild!!)
                     datastore.save(profile)
+                    // check the selectedContributor have the role
+                    if (event.guild!!.getMemberById(it)!!.roles.contains(event.guild!!.getRoleById(System.getenv("PING_ID"))!!)) {
```

Comment:
I thought we weren't using roles?

Reply:
> I thought we weren't using roles?

I think he's still using roles as that's what Stephen outlined in #47

Reply:
@Zcamt ^

Reply:
@sttephen thoughts? I think it would be a lot cleaner to just keep this in profile data

Reply:
> @sttephen thoughts? I think it would be a lot cleaner to just keep this in profile data

Think Stephen mainly suggested using a role to make it a more simple issue to resolve for someone
But I agree with it being cleaner to have it in the profile data